gui/src/components/UnknownDeviceModal.tsx

```diff
@@ -28,7 +29,7 @@ export function UnknownDeviceModal() {
         ['/onboarding/connect-trackers', '/settings/firmware-tool'].includes(
           pathname
         ) ||
-        state.ignoredTrackers.has(macAddress as string) ||
+        ignoredTrackers.has(macAddress as string) ||
```

Comment:
not needed if ignoredTrackers is `Set<string>`
```suggestion
        ignoredTrackers.has(macAddress) ||
```